drivers/docker/driver.go

```diff
@@ -1027,10 +1027,7 @@ func (d *Driver) DestroyTask(taskID string, force bool) error {
 		h.logger.Warn("failed to stop container during destroy", "error", err)
 	}
 
-	if err := h.dlogger.Stop(); err != nil {
-		h.logger.Error("failed to stop docker logger process during destroy",
-			"error", err, "logger_pid", h.dloggerPluginClient.ReattachConfig().Pid)
-	}
+	defer h.shutdownLogger()
```

Comment:
Do we need this here in addition to in the handle?

Reply:
I'm mostly being paranoid, I don't think so tho because it goes based on WaitContainer() - Will remove it here tho.